src/Microsoft.VisualStudio.Threading.Analyzers/VSTHRD010VsServiceUsageAnalyzer.cs

```diff
@@ -119,6 +119,7 @@ public override void Initialize(AnalysisContext context)
         private static bool IsVisualStudioShellInteropAssembly(string assemblyName)
         {
             return assemblyName.StartsWith("Microsoft.VisualStudio.Shell.Interop", StringComparison.OrdinalIgnoreCase)
+                || assemblyName.Equals("Microsoft.VisualStudio.OLE.Interop", StringComparison.OrdinalIgnoreCase)
```

Comment:
Should this list contain all the Microsoft.VisualStudio.*.Interop assemblies?

I found these other assemblies too:
Microsoft.VisualStudio.WCFReference.Interop
Microsoft.VisualStudio.CodeAnalysis.Interop
Microsoft.VisualStudio.Converters.Interop
Microsoft.VisualStudio.Coverage.Interop
Microsoft.VisualStudio.Data.Interop
Microsoft.VisualStudio.DataTools.Interop
Microsoft.VisualStudio.Debugger.InteropA
Microsoft.VisualStudio.Debugger.Interop
Microsoft.VisualStudio.Debugger.SymbolLocator.Interop
Microsoft.VisualStudio.DesignTools.UwpInterop
Microsoft.VisualStudio.Platform.VSEditor.Interop
Microsoft.VisualStudio.Imaging.Interop
Microsoft.VisualStudio.Debugger.Interop
Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.Interop
Microsoft.VisualStudio.TextManager.Interop